hoprd/rest-api/src/lib.rs

```diff
@@ -322,7 +322,7 @@ async fn build_api(
                 .route("/node/peers", get(node::peers))
                 .route("/node/entryNodes", get(node::entry_nodes))
                 .route("/node/metrics", get(node::metrics))
-                .route("/node/graph", get(node::channel_graph))
+                .route("/node/graph", post(node::channel_graph))
```

Comment:
Why `POST`? The operation is to `GET` data, not push it into the node.

Reply:
`GET` cannot have request body to pass the config for the graph export through.